### PR TITLE
Added two tests

### PR DESCRIPTION
**Pull request by AlexDoh**
Added two tests from codefroces

### CODE REVIEW

File: entry-test/coverByDominoes.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use.strict';
+
+function coverByDominoes(boardHeight, boardWidth){
+
+    let boardSquare = boardHeight * boardWidth;
+
+    return Math.floor(boardSquare / 2);
+};
```
Comment (pavlom0x) [inline entry-test/coverByDominoes.js:8]: I think you should read task description again.
You should have three input value *boardHeight*, *boardWidth* and *DominoseHeight*

File: entry-test/nextRound.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use.strict';
+
+function nextRound(allParticipants, lowestPlaceForNextRound, arr){
```
Comment (pavlom0x) [inline entry-test/nextRound.js:3]: I can't understand what the task it is
provide me with a link for description

**Review by pavlom0x — CHANGES_REQUESTED** [review]
Description very general
you should describe in pull request what functionality you added
mention what it should solve/do

### Example

Added methods for solving tasks
- watermelon
- nextYearFind
- tests for methods

File: entry-test/coverByDominoes.js (hunk 1 above)
Comment (AlexDoh) [reply entry-test/coverByDominoes.js:8]: http://codeforces.com/problemset/problem/50/A

Size of dominoes are standart - 2x1

File: entry-test/nextRound.js (hunk 2 above)
Comment (AlexDoh) [reply entry-test/nextRound.js:3]: http://codeforces.com/problemset/problem/158/A

**Pull request merged**